### ISSUE TITLE
✔️ 🐞 Bug report: bug in container

### ISSUE THREAD

**Issue by Michaelgimii** [elementor/elementor#19551]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I tried to create [this design](https://prnt.sc/YJ8OXI5hmvN8) using [these settings](https://prnt.sc/vrLR1eKF-5LL). In fact, here we have a container that is full width with a content width of 1200px. So our container will be in the center of the page. But the problem here is that when we space the container, the container gets off center and sticks to the left side. Please watch this video @shilo-ey : 

https://user-images.githubusercontent.com/88034902/185807050-79161dc5-e34e-48d1-aeb7-b87188e52aef.mp4

**As you can see in the video above, after I changed the container spacing, the container stuck to the left. While it should be in the center.**

### Steps to reproduce

-

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

-

**Comment by perks98** [comment]
@Michaelgimii for the container to center without another nested container it uses margin auto, changing the margin value resets that so it goes to the left. To do what you want currently, which is use margin for your own spacing, you would need to use custom CSS. Like using the CSS in the dropdown below in the page custom CSS area.

<details>
<summary>
<b>CSS and Screenshot</b>
</summary>

```
selector .elementor {
    display: flex;
    flex-direction: column;
    align-items: center;
}
```

![Screenshot 2022-08-22 200333](https://user-images.githubusercontent.com/55829113/185999294-59e56bb0-4412-41b0-84ed-95f03ab72046.png)
*use at own risk
</details>

**Comment by nicholaszein** [comment]
### This issue has been resolved in Elementor **v3.8**! 🥳 

✔️ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂
